nevergrad/optimization/test_optimizerlib.py

```diff
@@ -197,6 +197,86 @@ def test_infnan(name: str) -> None:
         result < 2.0, f"{name} failed and got {result} with {recom.value} (type is {type(optim)})."
 
 
+def suggestable(name: str) -> bool:
+    # Some methods are not good with suggestions.
+    if "TBPSA" in name:
+        return False
+    if "EMNA" in name or "EDA" in name:
+        return False
+    if "Stupid" in name:
+        return False
+    if "BO" in name:
+        return False
+    if "Pymoo" in name:
+        return False
+    return True
```

Comment:
```suggestion
    keywords = ["TBPSA", "BO", "EMNA", "EDA", "BO", "Stupid", "Pymoo"]
    return not any(x in name for x in keywords)
```

I still don't like relying on names though, would be better to use the base classes, or rely on TellNotAskedNotSupported